content/en/docs/languages/cpp/callback.md

```diff
@@ -0,0 +1,856 @@
+---
+title: Asynchronous Callback API Tutorial
+linkTitle: Asynchronous Callback API Tutorial
+weight: 60
+spelling: cSpell:ignore classgrpc Impl's
+---
+
+This tutorial shows you how to write a simple server and client in C++ using
+gRPC's asynchronous callback APIs. The example used in this tutorial follows the
+[RouteGuide example](https://github.com/grpc/grpc/tree/{{< param grpc_vers.core >}}/examples/cpp/route_guide).
+
+## Overview
+
+gRPC C++ offers two kinds of APIs: sync APIs and async APIs. More specifically,
+we have two kinds of async APIs: the old one is completion-queue based; the new
+one is callback-based, which is easier to use. In this codelab, we will focus on
```

Comment:
Do we use this term `codelab` for OSS?

Reply:
and I thought I had done a sanity check :D
Thanks for pointing it out.